yhondanilo/chess.rb

```diff
@@ -0,0 +1,47 @@
+class Chess
+  attr_accessor :fila, :columna
+
+  def initialize(fila, columna)
+    @fila, @columna=fila, columna
+    intercalar
+  end
+  def intercalar
+    @board=Array.new(@fila){Array.new(@columna)}
+    @board[0][0]=1
+    for f in 0..self.fila-1
+      for c in 0..self.columna-1
+        if @board[f][c-1]==0
+          @board[f][c]=1
+        elsif @board[f][c-1]==1
+          @board[f][c]=0
+        end
+        #if @board[f][c]==@board[f][@columna-1]
+         if c==@columna-1
```

Comment:
Inconsistent indentation detected.<br>Surrounding space missing for operator '=='.<br>Surrounding space missing for operator '-'.